core/BUILD.bazel

```diff
@@ -11,6 +11,19 @@ cc_library(
     visibility = ["//visibility:public"],
 )
 
+cc_library(
+    name = "core_testutils",
```

Comment:
Probably change the library name to core_test_utils? Both here an in CMake.

Reply:
Done!